routes/accounts.js

```diff
@@ -8,7 +8,7 @@ const validateEmail = require( '../src/mail/validate_email' )
 
 
 const AUTH_OPTIONS = {
-  successRedirect: '/items',
+  successRedirect: '/items/weekly',
```

Comment:
This is a very surprising change - why does authentication change to a weekly route?